JobConfig/primary/StopParticle.fcl

```diff
@@ -20,6 +20,7 @@ physics : {
   filters : {
     @table::Common.filters
     @table::Primary.filters
+    @table::genFilters.filters
```

Comment:
Where is genFilters defined?   I don't see it in this PR.
I suggest instead to add the definition of the filters needed for primary generation to Primary.filters (in primary/prolog), and instantiate the specific genfilter you need in the specfic script.

Reply:
probably in Offline EventGenerator/fcl